corehq/apps/smsbillables/async_handlers.py

```diff
@@ -103,3 +105,46 @@ def _fmt_success(self, response):
             } for r in response]
         }, cls=LazyEncoder)
         return success
+
+
+class PublicSMSRatesAsyncHandler(BaseAsyncHandler):
+    slug = 'public_sms_rate_calc'
+    allowed_actions = 'public_rate'
+
+    @property
+    def public_rate_response(self):
+        return self.get_rate_table(self.data.get('country_code'))
+
+    @quickcache(['country_code'], timeout=24 * 60 * 60)
+    def get_rate_table(self, country_code):
+        backends = SMSBackend.view(
```

Comment:
@nickpell I think there's just one backend that I need to remove (I need to confirm) but when I remove it, it will drop out of this view